pkgs/development/libraries/xcb-util-cursor/default.nix

```diff
@@ -1,12 +1,13 @@
-{ stdenv, fetchurl, bashInteractive, autoconf, automake, libtool, pkgconfig
+{ stdenv, fetchgit, bashInteractive, autoconf, automake, libtool, pkgconfig
 , git, xlibs, gnum4, libxcb, gperf }:
 
 stdenv.mkDerivation rec {
   name = "xcb-util-cursor-0.1.1";
```

Comment:
We'll need to update the version string here. In fact, it would probably be better to make a separate nix expression for `xcb-util-cursor-HEAD` and only feed that into `i3` on darwin.

(This is assuming that 0.1.1 is the latest release)